libbpf-rs/tests/test.rs

```diff
@@ -156,6 +157,57 @@ fn test_object_map_lookup_flags() {
         .is_err());
 }
 
+#[test]
+fn test_object_map_key_iter() {
+    bump_rlimit_mlock();
+
+    let mut obj = get_test_object();
```

Comment:
Note that the changes introduced in #51 will require `get_test_object()` to be changed to `get_test_object("runqslower.bpf.o")`